msteams-platform/resources/dev-preview/developer-preview-features.md

```diff
@@ -9,6 +9,10 @@ keywords: teams preview developer features
 
 The developer preview includes the following new features:
 
+## App customization
+
+As a Teams admin, you can now customize your apps based on your organization's need. Use the [app customization feature](~/concepts/design/design-teams-app-overview.md) to rebrand the apps.
+
```

Comment:
Suggest pivoting this from the developer's standpoint. Suggest: 'As a Teams Developer, you can now define a select set of properties ( see [app customization feature](~/concepts/design/design-teams-app-overview.md)) which a Teams admin can customize/rebrand based on their organizations need